library/Booster/Pattern/Bool.hs

```diff
@@ -39,7 +43,7 @@ pattern AndBool l r =
                 []
                 [SortBool, SortBool]
                 SortBool
-                (SymbolAttributes TotalFunction IsNotIdem IsNotAssoc IsNotMacroOrAlias CanBeEvaluated Nothing)
+                TotalFunctionAttributes
```

Comment:
These should all have smt hooks, e.g. `smt-hook(and)`. perhaps we should use the QQ for these and paste in the kore defintion